tests/test_libraries.py

```diff
@@ -0,0 +1,128 @@
+from nose.tools import eq_
+
+import js_helper
+import validator.testcases.content as test_content
+from tests.test_frameworks import disposable_metadata
+from validator.compat import FX9_DEFINITION
+from validator.errorbundler import ErrorBundle
+from validator.testcases import libraries
+from validator.xpi import XPIManager
+
+
+def test_whitelisted_files():
+    """
+    Tests the validator's ability to hash each individual file and (based on
+    this information) determine whether the addon passes or fails the
+    validation process.
+    """
+
+    package_data = open("tests/resources/libraryblacklist/blocked.xpi")
+    package = XPIManager(package_data, mode="r", name="blocked.xpi")
+    err = ErrorBundle()
+
+    test_content.test_packed_packages(err, package)
```

Comment:
I would create a helper function for the 4 lines above that seem to be used throughout this file. Also, I'd use the `with open(...) as ...` idiom.